src/js/utils/JobUtil.js

```diff
@@ -90,6 +90,9 @@ const JobUtil = {
 
     let [schedule = {}] = job.getSchedules();
     if (schedule.id != null && schedule.cron != null) {
+      if (schedule.concurrencyPolicy == null) {
+        schedule.concurrencyPolicy = 'ALLOW';
```

Comment:
But one question: shouldn't serverside validation itself fail if the user has not provided one value? Or shouldn't it add a default for us?